postgis/postgis.sql.in

```diff
@@ -3550,6 +3550,19 @@ CREATE OR REPLACE FUNCTION ST_FrechetDistance(geom1 geometry, geom2 geometry, fl
 	LANGUAGE 'c' IMMUTABLE STRICT PARALLEL SAFE
 	_COST_HIGH;
 
+-- Availability: 2.4.0
+CREATE OR REPLACE FUNCTION ST_FrechetDistance(geom1 geometry, geom2 geometry, float8 default -1)
```

Comment:
This function has been duplicated, I imagine by error

Reply:
oops